test_files/jsdoc/jsdoc.ts

```diff
@@ -143,3 +143,11 @@ const DEFINE_WITH_INFERRED_TYPE = false;
  * @define
  */
 const DEFINE_WITH_DECLARED_TYPE: string = 'y';
+
+/**
```

Comment:
Can you move this to a new test? `jsdoc.ts` is overflowing, making it hard to debug specific issues.

Also, can you include a `/** @fileoverview */` comment in that new test that documents what's being tested by the code sample?

Reply:
Done, added the new test in `test_files/jsdoc`, because that still seems like the right place for it.